services/web/server/tests/integration/computation/conftest.py

```diff
@@ -46,7 +46,7 @@ async def logged_user(client, user_role: UserRole):
 
 
 @pytest.fixture
-async def user_project(client, mock_project, logged_user):
+async def user_project(client, mock_project, logged_user) -> NewProject:
```

Comment:
It actually returns a ``Dict`` not ``NewProject``. See [implementation](https://github.com/ITISFoundation/osparc-simcore/blob/9143165172586f205d0fae63aa2c4ebffa79428b/packages/pytest-simcore/src/pytest_simcore/helpers/utils_projects.py#L123) of with ``__aenter__``

Reply:
yep and the one above as well. corrected!